charts/argo/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: v2.11.3
+appVersion: v2.11.7
```

Comment:
If we are bumping the appVersion, I think we should bump `image.tag` in values.yaml too.

Reply:
Done.